src/gameplay/AllPlayersScreen.js

```diff
@@ -61,7 +64,19 @@ function PlayerScreen(props) {
 		return () => subscribe();
 	}, [currentTurn]);
 
-	if(outOfGame){
+	if (winner) {
+		return (
+			<div>
+				<h3>{winner} is the winner!</h3>
+				<p style={{paddingTop: "1em"}}>Thanks for playing MVP Coup!</p>
+				<div style={{paddingBottom: "1em", paddingTop: "1em"}}>
+					<Button type="button" className="btn btn-lg btn-light" style = {{width:"20em"}} onClick={
+						() => (<Redirect to="/" />)
```

Comment:
does it need return in there somewhere?

Reply:
if you do a one liner then you don't need `return` (I think maybe you can't redirect from just any random component, but idk)